helm_deploy/create-and-vary-a-licence-api/templates/app-jobs.yaml

```diff
@@ -29,7 +29,7 @@ spec:
               args:
                 - /bin/sh
                 - -c
-                - curl --fail {{- if .allowRetry }} --retry 5 {{ end }} -X POST "https://$BASE_URL/jobs/{{ .path }}"
+                - curl --fail {{- if .allowRetry }} --retry 5 {{ end }} -X POST "https://$BASE_URL/jobs{{ .path }}"
```

Comment:
I'd updated this to not include the `/jobs` bit but forgotten to update the actual values files to remove it